modular_chomp/code/game/objects/mob_spawner.dm

```diff
@@ -9,18 +9,32 @@
 	anchored = 1
 	invisibility = 101
 	spawn_types = list(
-		/mob/living/simple_mob/mechanical/hivebot/swarm = 1,
-		/mob/living/simple_mob/animal/space/carp = 10,
-		/obj/structure/closet/crate/mimic/dangerous = 2,
-		/mob/living/simple_mob/animal/space/bats = 70,
-		/mob/living/simple_mob/vore/jelly = 25,
+		/mob/living/simple_mob/vore/bat = 70,
```

Comment:
Oh, you'll still have bats, they just won't be bat _swarms_ anymore. Just really big nommy bats.